src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -18,12 +18,68 @@
             <h1 class="govuk-heading-m">
                 @title
             </h1>
-            <table id="pcs-details-table" class="govuk-table">
+            <table id="pcs-contact-details-table" class="govuk-table">
                 <tbody class="govuk-table__body">
-                   
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">First Name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactData.FirstName</td>
```

Comment:
@stevelee-civica  This should the be current version of the contact details which is on the ContactData property